### ISSUE TITLE
what is this library?

### ISSUE THREAD

**Issue by mcheung63** [mulab/nV#1]
what is this library?

**Comment by cgcgbcbc** [comment]
This project is part of an open source Computer Algebra System "mathmu", which aims at scientific computing, like Matlab, Mathematica. But since most of the author graduated from school, this project was no longer under active developing. The project is "open-sourced" here in case that if it will help someone who is interested at scientific computing related study.

Project short description can be found in the following articles.
http://www.tsinghua.edu.cn/publish/news/4205/2011/20110225232541875190560/20110225232541875190560_.html

**Comment by zmy** [comment]
Yep, we tried to build an open source alternative of Mathematica and successfully built some basic components like interpreter, polynomial module, constant algorithms, calculus, etc. But without enough spare time and funding we sadly decided to stop the project.

Feel free to explore through the deprecated code which is full of our memories and efforts. Good luck and have fun!

**Comment by mcheung63** [comment]
你是陳渝的學生嗎

Thanks
From Peter

**Comment by mcheung63** [comment]
你是陳渝的學生嗎

Thanks
From Peter
